docs/Configuration.md

```diff
@@ -0,0 +1 @@
+# Configuration

```

Comment:
For the configuration guide, let's move the content from the exporter READMEs into one place under a an L2 header "Configuring Exporters". That way we can reduce the repetitive content in each directory.